### ISSUE TITLE
User Defined Language broken XML with v7.6.6

### ISSUE THREAD

**Issue by jensensen** [notepad-plus-plus/notepad-plus-plus#5501]
### Description of the Issue
Any XML corrupted

Import and export also fails.
With v7.6 everything is OK.

### Steps to Reproduce the Issue

1. Export UDL or
2. Import UDL
3. USER\AppData\Roaming\Notepad++\userDefineLang.xml
4. Version 32bit / 64 bit anyway

### Expected Behavior
get clean files but

### Actual Behavior
any xml file got errors, when udl has been imported or exported.

**Comment by MetaChuh** [comment]
@jensensen 
i can not reproduce this behaviour when importing or exporting udls in notepad++ 7.6.6

if you have a specific udl which you can't export, please make a zip of your `userDefineLang.xml` file and upload it here by clicking on the `attach files`bar below.

if you have a specific udl which you can not import, please try to copy that .xml file to the folder `%AppData%\Notepad++\userDefineLangs` and restart notepad++.
if it does not show up correctly in notepad++, please also zip and upload your file, and additionally provide us with your debug info.

please also try to go to `settings > preferences > default directory` and disable `use new style dialog`.
disabling new style dialog will automatically add the .xml file extension when exporting an udl.

-----

**please also test the following, to verify, if a clean notepad++ 7.6.6 would behave differently.**

* download the notepad++ 7.6.6. portable version from [>>> here <<<](https://notepad-plus-plus.org/repository/7.x/7.6.6/npp.7.6.6.bin.zip).
(the portable version does not require any installation and runs completely independent and isolated from your installed version. all custom settings, as well as any stuff you might want to try at the portable version, will not modify your installed version)

* extract `npp.7.6.6.bin.zip` to e.g. your desktop.

* **important note:** make sure to close all instances of notepad++ that might be running, before starting the portable version at the next step, to make sure you are using this portable version.

* open the extracted `npp.7.6.6.bin` folder and start this portable version by double-clicking on `notepad++.exe` inside this folder.
(note: notepad++.exe will just be seen as notepad++, if you have enabled to hide all known file extensions at your explorer settings)

* retry exporting and importing udl files at this portable version.

-----
```
general notes:

please close your issue as soon as you are satisfied,
to help keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and first
search if a related topics exists. you are welcome to post at either an eventually similar
topic, or create a new topic at Help Wanted or General Discussion of your choice.
```
[<kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd>](https://notepad-plus-plus.org/community/)
